Sources/MHCStudyDefinitionExporter/Study.swift

```diff
@@ -100,7 +100,22 @@ let mhcStudyDefinition = StudyDefinition(
                     .bodyMassIndex,
                     .leanBodyMass,
                     .bodyFatPercentage,
-                    .waistCircumference
+                    .waistCircumference,
```

Comment:
Narayan mentioned this yesterday in the meeting: We should also add `numberOfTimesFallen` as well 👍